proto/core.proto

```diff
@@ -36,6 +36,13 @@ enum ErrorType {
   APP_NOT_ACTIVE = 3;
 };
 
+enum SeedGenerationStatus {
+  SEED_GENERATION_STATUS_INIT = 0;
+  SEED_GENERATION_STATUS_PASSPHRASE = 1;
+  SEED_GENERATION_STATUS_PIN = 2;
```

Comment:
Please verify if the status for the pin is required or not as the cysync displays it as "enter pin and tap card".

Reply:
I don't think `PIN` step is required here

Reply:
Removed